gratipay/models/participant.py

```diff
@@ -380,10 +380,11 @@ def add_email(self, email, resend_threshold='3 minutes'):
 
         # Check that this address isn't already verified
         owner = self.db.one("""
-            SELECT participant
-              FROM emails
-             WHERE address = %(email)s
-               AND verified IS true
+            SELECT p.username
+              FROM emails e INNER JOIN participants p
+                ON e.participant_id = p.id
+              WHERE e.address = %(email)s
```

Comment:
ok so that's what the uneven indentation at the start of the line is about! I was scratching my head with that. Man, now you're just messing with a java guy. :smirk: